rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPNodeRepository.scala

```diff
@@ -99,7 +99,14 @@ class WoLDAPNodeRepository(
    * what is given in paramater
    */
   def updateNodeProperties(nodeId: NodeId, properties: Seq[NodeProperty], modId: ModificationId, actor:EventActor, reason:Option[String]) : Box[Node] = {
-    val updateNode = (node: Node) => node.copy(properties = properties)
+    val updateNode = (node: Node) => {
+      val newProperties = properties.groupBy(_.name)
+      val oldProperties = node.properties.groupBy(_.name)
+      // Merge all properties, since this a Map old ones will be overwritten by new since they have the same key
+      // Filter to remove properties with empty values
+      val allProperties = (oldProperties ++ newProperties).values.flatten.toSeq.filterNot(_.value == "")
```

Comment:
If we keep up with the same logic we should rename the function 'setNodeProperties', I found it more logic with the behavior (with update i did not feel it would overwrite the current params)